### PR TITLE
MM5.1

### PR DESCRIPTION
**Pull request by mlhatfield**

### CODE REVIEW

File: jobs/mattermost/templates/config/initial-config.json.erb [hunk 1]
```diff
@@ -190,7 +201,8 @@
         "UseChannelInEmailNotifications": <%= p("mattermost.EmailSettings.UseChannelInEmailNotifications") %>,
         "LoginButtonColor": "<%= p("mattermost.EmailSettings.LoginButtonColor") %>",
         "LoginButtonBorderColor": "<%= p("mattermost.EmailSettings.LoginButtonBorderColor") %>",
-        "LoginButtonTextColor": "<%= p("mattermost.EmailSettings.LoginButtonTextColor") %>"
+        "LoginButtonTextColor": "<%= p("mattermost.EmailSettings.LoginButtonTextColor") %>",
```
Comment (tejaycar) [inline jobs/mattermost/templates/config/initial-config.json.erb]: seems we have this twice.   does that seem right to you?  see line 280

**Comment by tejaycar** [comment]
Looks good to me Mike

File: jobs/mattermost/templates/config/initial-config.json.erb (hunk 1 above)
Comment (mlhatfield) [reply jobs/mattermost/templates/config/initial-config.json.erb]: There should be a `LoginButtonTextColor` attribute for both EmailSettings and LdapSettings - is that what you're talking about Tejay?

Comment (tejaycar) [reply jobs/mattermost/templates/config/initial-config.json.erb]: It sure is.  I was hoping it was something like that.  Sounds like we're fine then.

**Comment by drnic** [comment]
@tejaycar @mlhatfield good to merge? (title is still "WIP:")

**Comment by mlhatfield** [comment]
@drnic I'm still unable to successfully get the load tests to run (Haven't been working for us since 4.8), working through it with MM support currently.

**Comment by mlhatfield** [comment]
@drnic This is ready to go

**Pull request merged**